README.markdown

```diff
@@ -1023,7 +938,7 @@ func updateCurrentUser(notification: Notification) {
 
       // Since `updateCurrentUser` is not guaranteed to be called on the main thread,
       // We must dispatch onto the main thread before calling `mt_` functions.
-      DispatchQueue.main.async{ 
+      DispatchQueue.main.async{
```

Comment:
nitpick: add space before `{` to conform with standards

Reply:
I removed these examples related to `mt_` since they no longer apply without the rules.